frontend/tasks/scripts.js

```diff
@@ -55,36 +61,72 @@ gulp.task('scripts-misc', () => {
 });
 
 gulp.task('scripts-generate-static-html', () => {
-  return commonBrowserify('generate-static-html.js', browserify({
+  return browserify({
     entries: [config.SRC_PATH + 'generate-static-html.js'],
     debug: config.IS_DEBUG,
     fullPaths: config.IS_DEBUG,
     transform: [babelify],
-    standalone: 'generate-static-html'
-  }).external(vendorModules));
+    standalone: 'generate-static-html',
+    bundleExternal: false
+  }).external(vendorModules)
+    .bundle()
+    .pipe(source('generate-static-html.js'))
+    .pipe(buffer())
+    .pipe(sourcemaps.init({loadMaps: true}))
+     // don't uglify in development. eases build chain debugging
+    .pipe(gulpif(!config.IS_DEBUG, uglify()))
+    .on('error', gutil.log)
+    .pipe(sourcemaps.write('./'))
+    .pipe(gulp.dest(config.DEST_PATH));
 });
 
-gulp.task('scripts-app-main', () => {
+gulp.task('scripts-app', () => {
   return commonBrowserify('app.js', browserify({
-    entries: [config.SRC_PATH + 'app/index.js'],
+    entries: [config.SRC_PATH + 'app.js'],
     debug: config.IS_DEBUG,
     fullPaths: config.IS_DEBUG,
-    transform: [babelify]
+    transform: [babelify],
+    standalone: 'app',
+    bundleExternal: false
   }).external(vendorModules));
 });
 
+gulp.task('scripts-app-individual', () => {
```

Comment:
Since [switching back to one app.js bundle](https://github.com/mozilla/testpilot/pull/2351/commits/9b339acb8a24b2923cee1e03d8bca2fd8a2c6f45), this task isn't needed anymore right?